railgun/pkg/constants/constants.go

```diff
@@ -0,0 +1,7 @@
+package constants
+
+const (
+	// KongIngressClass is the string value needed by default to configure the annotation "kubernetes.io/ingress.class"
+	// which indicates which ingress controller is responsible for an Ingress resource.
+	KongIngressClass = "kong"
```

Comment:
This is a CLI flag in KIC 1.x and needs to remain as such.

Reply:
Sounds good, context added in https://github.com/Kong/kubernetes-ingress-controller/issues/1097